src/DynamoCore/Models/HomeWorkspaceModel.cs

```diff
@@ -131,6 +131,7 @@ public virtual void OnSetNodeDeltaState(DeltaComputeStateEventArgs e)
                 engine,
                 scheduler,
                 factory,
+                Enumerable.Empty<PresetModel>(),
```

Comment:
Again, let's move this to follow `IEnumerable<AnnotationModel>` in the constructor.